src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage,
+  closeError: () => void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  closeError,
+}) => {
+  useEffect(() => {
+    setTimeout(() => {
+      closeError();
+    }, 3000);
```

Comment:
You should clear all timers on the component unmount to avoid unpredictable side effects https://bobbyhadz.com/blog/react-cleartimeout